binary/expression.proto

```diff
@@ -59,6 +59,17 @@ message Expression {
             List list = 30;
         }
 
+        message Decimal {
+            // little-endian twos-complement integer representation of complete value (ignoring precision)
+            // Always 16 bytes in length
+            bytes value = 1;
+            // declared scale of decimal literal
+            int32 scale = 2;
```

Comment:
Minor nit here: do you mind putting precision first? We typically list it that way (and it helps me remember which is which, which I always forget).